src/features/query-bar/components/fixtures.js

```diff
@@ -3,7 +3,7 @@ import type { ColumnType, ConditionType } from '../flowTypes';
 
 const initialCondition: ConditionType = {
     id: 0,
-    columnDisplayText: '',
+    columnDisplayText: 'Contract Value',
```

Comment:
The reason that I set 'Contract Value' here is that in the snapshot, I want to render a column that has an options property that is not null.